### DISCUSSION TITLE
Long Health Bars hidden in files.

### DISCUSSION THREAD

**Discussion by Piggymaster1222** [JordanSantiagoYT/FNF-JS-Engine#710] [General]
I did find the long health bar png'
![Screenshot 2024-11-01 130705](https://github.com/user-attachments/assets/3b6888c9-edab-435d-8cd3-66abb2081600)
s in the files. and I was wondering when they will be added back to JS Engine. maybe in 1.38.0?

**Comment by moxie-coder** [comment]
> I did find the long health bar png' ![Screenshot 2024-11-01 130705](https://private-user-images.githubusercontent.com/179584620/382351895-3b6888c9-edab-435d-8cd3-66abb2081600.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzA0ODExOTQsIm5iZiI6MTczMDQ4MDg5NCwicGF0aCI6Ii8xNzk1ODQ2MjAvMzgyMzUxODk1LTNiNjg4OGM5LWVkYWItNDM1ZC04Y2QzLTY2YWJiMjA4MTYwMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTAxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTEwMVQxNzA4MTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01ZDdhMzE4YzVjNThjMzY3ZGNmZDVlMDE5YmMwMWRkMjBmZTZkMWMyMTdmYTVkZDc5ZTgxMjA5NWJhNzU3Yjk1JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.qbZcdg2zoK3AFSYF4Mpya9KEiJlZ4iRYhB920mAzH5I) s in the files. and I was wondering when they will be added back to JS Engine. maybe in 1.38.0?

there’s no plan on readding them, since you can easily do it in lua

**Reply by Piggymaster1222** [reply]
ok
